lib/travis/build/addons/phantomjs.rb

```diff
@@ -0,0 +1,72 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class PhantomJs < Base
+        SUPER_USER_SAFE = true
+
+        attr_reader :version
+
+        def after_prepare
+          sh.fold 'install_phantomjs' do
+            sanitize(raw_version)
+
+            unless version
+              sh.echo "Invalid version '#{raw_version}' given. Only 1.9.8 and 2.0.0 are supported at the moment", ansi: :red
+              return
+            end
+
+            sh.echo "Installing PhantomJS #{version}", ansi: :yellow
```

Comment:
We should check the current version with `phantomjs --version` (which takes about a whole second) and bail out of this addon if it matches the desired one.